admin_guide/upgrading_citus.rst

```diff
@@ -49,16 +49,16 @@ Step 1. Update Citus Package
 .. code-block:: bash
 
   sudo apt-get update
-  sudo apt-get install postgresql-9.6-citus-7.5
+  sudo apt-get install postgresql-11-citus-8.0
   sudo service postgresql restart
 
 **Fedora, CentOS, or Red Hat**
 
 .. code-block:: bash
 
   # Fedora, CentOS, or Red Hat
-  sudo yum swap citus62_96 citus75_96
-  sudo service postgresql-9.6 restart
+  sudo yum swap citus75_11 citus80_11
```

Comment:
We need to replace 11 with 10 here. An upgrade from 7.5 to 8.0 will need to be on PG10. There aren't packages generated for 7.5 and 11. So `citus75_11` doesn't exist and will error out.